documentation/release_notes.markdown

```diff
@@ -61,179 +94,253 @@ This release retires all api versions before v4 and promotes v4 to be
-      standards.
-    * Lots of template files were transferred into resource/ext, this is what ezbake
-      expects.
-    * Documentation has moved inside /usr/share/puppetdb/docs to a new location
-      depending on the distribution. This is to follow the ezbake standards.
-    * Many symlinks were removed from /usr/share/puppetdb that were pointing
-      at the /var/lib/puppetdb and /etc/puppetdb/ spaces. This finishes a partially
-      completed migration from the past.
+    Installing via the package will now no longer default to using HyperSQL.
+    Instead now we expect PostgreSQL by default, and will return an error if
+    it hasn't been configured.
 
 ### New Features
 
-* Add support for hyphenated classnames, supported by Puppet 2.7.x - 3.7.x. ([PDB-1024](https://tickets.puppetlabs.com/browse/PDB-1024))
+* We now store report metrics ([PDB-1192](https://tickets.puppetlabs.com/browse/PDB-1192))
 
-* Add a '-v' flag to the CLI to get the PuppetDB version.
+* We now store report logs ([PDB-1192](https://tickets.puppetlabs.com/browse/PDB-1184))
 
-* The `reports` database table was using the hash string as its
-  primary key, we know have switched to using a smaller bigint primary
-  key for that table, which should result in faster joins in most
-  cases, and hopefully smaller foreign key indexes sizes for tables
-  relating to
-  `reports`. ([PDB-1218](https://tickets.puppetlabs.com/browse/PDB-1218))
+* We have added `count` aggregation capabilities to the query API ([PDB-1181](https://tickets.puppetlabs.com/browse/PDB-1181))
 
-* The reports endpoint now implements the `latest_report?` query, which
-  filters the response to only include reports that are from the
-  latest puppet run for each node.
-  ([PDB-1244](https://tickets.puppetlabs.com/browse/PDB-1244))
+* Support `extract` as a top level query operator ([PDB-207](https://tickets.puppetlabs.com/browse/PDB-207))
 
-* We have dropped the latest_reports table in favor of storing a reference to
-  the latest report of each certname in the certnames table.
-  ([PDB-1254](https://tickets.puppetlabs.com/browse/PDB-1254))
+    This provides support for requesting only the fields that you need in
+    an API response when querying data.
 
-* The load testing tool can now optionally use a default set of testing data or
-derive data from a PuppetDB export tarball.
+* Support fallback PuppetDB connections ([PDB-100](https://tickets.puppetlabs.com/browse/PDB-100))
 
-* We have now added [edges](./api/query/v4/edges.html) querying capability to the top-level namespace. You can also query edges specific to a particular catalog using the new `edges` suffix on the [catalogs](./api/query/v4/catalogs.html) endpoint. ([PDB-1228](https://tickets.puppetlabs.com/browse/PDB-954)).
+    When configuring your connection to PuppetDB, you can now specify a fallback.
+    Instead of the old `server` and `port` config, you can now use a
```

Comment:
I think we should recommend switching to the new version since the old `server` and `port` is deprecated.